solutions/access-management/variables.tf

```diff
@@ -0,0 +1,1553 @@
+}
+
+variable "admin_observability_ag_service_ids" {
+  type        = list(string)
+  description = "A list of Service IDs that you want to add to the admin observability access group"
+  default     = []
+}
+
+##############################################################################
+# Admin Security Services Variables
+##############################################################################
+
+variable "admin_security_ag_name" {
+  type        = string
+  description = "Name of the Administrator Security access group"
+  default     = "admin-security-group"
+}
+
+variable "admin_security_ag_policies" {
+  type = map(object({
+    roles              = list(string)
+    account_management = optional(bool)
+    tags               = set(string)
+    resources = optional(list(object({
+      region               = optional(string)
+      attributes           = optional(map(string))
+      service              = optional(string)
+      resource_instance_id = optional(string)
+      resource_type        = optional(string)
+      resource             = optional(string)
+      resource_group_id    = optional(string)
+    })))
+    resource_attributes = optional(list(object({
+      name     = string
+      value    = string
+      operator = optional(string)
+    })))
+  }))
+  description = "A map of policies for the Administrator Security access group, has a set of default policies that can be overridden"
+  default = {
+    "hs-crypto" = {
+      roles = ["Administrator", "Manager", "Certificate Manager", "Vault Administrator", "Key Custodian - Creator", "Key Custodian - Deployer", "KMS Key Purge Role"]
+      resources = [{
+        service = "hs-crypto"
+      }]
+      tags = []
+    }
+    "kms" = {
+      roles = ["Manager"]
+      resources = [{
+        service = "kms"
+      }]
+      tags = []
+    }
+    "compliance" = {
+      roles = ["Administrator", "Manager"]
+      resources = [{
+        service = "compliance"
+      }]
+      tags = []
+    }
+    "secrets-manager" = {
+      roles = ["Administrator", "Manager"]
+      resources = [{
+        service = "secrets-manager"
+      }]
+      tags = []
+    }
+  }
+}
+
+variable "admin_security_ag_dynamic_rules" {
+  type = map(object({
+    expiration        = number
+    identity_provider = string
+    conditions = list(object({
+      claim    = string
+      operator = string
+      value    = string
+    }))
+  }))
+  description = "A map of dynamic rules for the admin security access group"
+  default     = {}
+}
+
+variable "admin_security_ag_add_members" {
+  type        = bool
+  description = "Enable this to add members to the admin security group"
+  default     = false
+}
+
+variable "admin_security_ag_description" {
+  type        = string
+  description = "Description of the admin security access group"
+  default     = null
+}
+
+variable "admin_security_ag_tags" {
+  type        = list(string)
+  description = "Tags that should be applied to the service"
```

Comment:
service?

Reply:
Description taken from https://github.com/terraform-ibm-modules/terraform-ibm-iam-access-group/blob/main/variables.tf#L34, unsure on what it should be changed to